source/quakedef.h

```diff
@@ -214,7 +216,13 @@ extern int psp_system_model;
 #include "psp/common.h"
 #include "psp/vid.h"
 #include "psp/sys.h"
-#endif // _3DS
+#elif __WII__
+#include <ogcsys.h>
+#include "wii/common.h"
+#include "wii/vid.h"
+#include "wii/sys.h"
+extern u32 MALLOC_MEM2;
```

Comment:
Put this in your `common.h`

Reply:
I'll move the `MALLOC_MEM2` define, but the rest is just following the standards that the other platforms are doing.